packages/client/src/pages/Admin/Dashboard/index.tsx

```diff
@@ -0,0 +1,117 @@
+import { IGameCreator, IGenre } from "interfaces/api";
+import { IGameForUI } from "interfaces/app";
+import React from "react";
+import { PieChart } from "react-minimal-pie-chart";
+import styles from "./styles.module.scss";
+import { config } from "config";
+
+interface DashboardProps {
+  genres: IGenre[];
+  games: IGameForUI[];
+  gameCreators: IGameCreator[];
+}
+
+export const Dashboard: React.FunctionComponent<DashboardProps> = ({
+  games,
+  genres,
+  gameCreators,
+}) => {
+  const labelConfig = {
+    fontSize: "9px",
+    fontFamily: "sans-serif",
+    fill: "#fff",
+  };
+  const shiftSize = 0.5;
+  return (
+    <div className={styles.dashboardContent}>
+      <h2 className={styles.header}>Dashboard</h2>
+      <div className={styles.diagramsBlock}>
+        <div className={`${styles.diagramContainer} ${styles.first}`}>
+          <h3 className={styles.header}>The most expensive games</h3>
+          <div className={styles.diagramContent}>
+            <PieChart
+              radius={PieChart.defaultProps.radius - shiftSize}
+              segmentsShift={() => shiftSize}
+              className={`${styles.diagram}`}
+              label={({ dataEntry }) => dataEntry.value}
+              labelStyle={{
+                ...labelConfig,
+                fontSize: "5px",
+              }}
+              data={games.map((game, i) => ({
+                title: game.name,
+                value: +game.price,
+                color:
+                  games.reduce((prev, curr) =>
+                    +prev.price < +curr.price ? curr : prev
+                  ).id === game.id
+                    ? config.colors.accent
+                    : config.colors.primary,
+              }))}
```

Comment:
Let's move calculations to the up, using some hooks. And then just pass prepared `data` into props.